apps/dashboard/src/utils/validation-schema.ts

```diff
@@ -12,26 +12,23 @@ import type { ContainerInStore } from "@/stores/container.store";
 
 const t = i18n.global.t;
 
-export const userDetailsSchema = toTypedSchema(
+export const createUserSchema =
     yup.object({
         firstName: yup.string().required(),
         lastName: yup.string().required(),
         email: yup.string().email(),
         nickname: yup.string(),
-        userType: yup.mixed().required(),
-        isActive: yup.boolean().required().default(true),
+        userType: yup.number().required().default(4),
         ofAge: yup.boolean().required().default(false),
         canGoIntoDebt: yup.boolean().required().default(false),
-    })
-);
+    });
```

Comment:
Since we moved to modules, would it not make more sense to keep this definition closer to its usage? maybe the parent component which handles the forms? Just some food for thought